cpanfile

```diff
@@ -92,8 +92,9 @@ requires 'XML::Simple';
 requires 'YAML';
 
 feature 'uk', 'FixMyStreet.com specific requirements' => sub {
-    # East Hampshire
+    # East Hampshire & Angus
     requires 'SOAP::Lite';
+    requires 'LWP::Protocol::https';
```

Comment:
I think with the more recent social login changes, this line shouldn't be necessary.